### DISCUSSION TITLE
Angular Gauge

### DISCUSSION THREAD

**Discussion by horaciodiez** [Live-Charts/LiveCharts2#521] [Q&A] [answered]
Hello! 
Thak you for your fantastic work.
I can´t finde the Angular Gauge in this version.
Is it available? If not, will it be?

Thank You again and Congratulations for this incredible product.

**Comment by k-ahmedhm** [comment]
Hello, why no one responded to this question ?

**Reply by alexachso** [reply]
https://lvcharts.com/#frameworks  Click on the platform of your liking. Then, if you scroll a bit you will see examples of gauges.

**Reply by horaciodiez** [reply]
Hi @alexachso, i saw the gauges. It is the "Angular" Gauge i am asking for.

**Reply by alexachso** [reply]
Hi @horaciodiez, Yes, my bad I taught you wanted a gauge with an angle to it like the example of the gauge with 270 degrees angle. The angular gauges you want seem not to be implemented yet. I am sure Alberto will add them eventually.

**Comment by beto-rodriguez** [comment, marked as answer]
It will be implemented, but it is not at this point.

**Reply by horaciodiez** [reply]
Hi Beto! 
Thank you.
